test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/DynamicSharedSymbols.java

```diff
@@ -90,7 +90,8 @@ private static void doTest(String topArchiveName) throws Exception {
         ProcessBuilder pb = new ProcessBuilder();
         pb.command(new String[] {JDKToolFinder.getJDKTool("jcmd"), Long.toString(pid), "VM.symboltable", "-verbose"});
         OutputAnalyzer output = CDSTestUtils.executeAndLog(pb, "jcmd-symboltable");
-        output.shouldContain("17 2: jdk/test/lib/apps\n");
+        output.shouldContain("17 3: jdk/test/lib/apps\n");  // 3 because as a TempSymbol will be found in the TempSymbolCleanupDelayer queue
+                   // note we might want to drain the queue before CDS dumps but this is correct for now, unless the queue length changes.
```

Comment:
nit typo: s/3 because as a TempSymbol/3 because a TempSymbol/

Probably should end L93 with a period and s/note/Note/ on L94.